src/Core/src/Credentials/CacheProvider.php

```diff
@@ -17,7 +17,7 @@
 final class CacheProvider implements CredentialProvider, ResetInterface
 {
     /**
-     * @var (null|Credentials)[]
+     * @var Credentials[]
```

Comment:
it's wrong, null could be an expected value

Reply:
Yes. I realised that now. Thank you